0.9/implementation-notes/index.html

```diff
@@ -8,6 +8,7 @@
             async class="remove"></script>
     <script class="remove">
         var respecConfig = {
+            publishDate: "2020-06-23",
```

Comment:
Not that it matters that much, but it appears that the release was June 29:
https://github.com/OCFL/spec/releases/tag/0.9

Reply:
that was because @ahankinson tagged after the fact.

Reply:
Yes, it was this commit https://github.com/OCFL/spec/commit/0124b825d3adafdc637d912fd5751544c00c8a61 -- which was June 23